tests/resource_management/libraries/script/script.py

```diff
@@ -8,4 +8,5 @@ def get_config():
         return {'configurations':
                 {'node.properties': {},
                  'jvm.config': {'jvm.config': ''},
-                 'config.properties': {}}}
+                 'config.properties': {}},
+                 'clusterHostInfo': {'presto_worker_hosts': [], 'presto_coordinator_hosts': []}}
```

Comment:
do we need coordinator_host as list? since Presto allows only one and may be we should also check that # coordinator == 1